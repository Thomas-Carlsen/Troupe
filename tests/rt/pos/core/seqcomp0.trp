(* positive; normal use of sequential composition *)

print "A"; print "B"; print "C"