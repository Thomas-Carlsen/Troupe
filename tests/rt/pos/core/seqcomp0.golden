2020-02-15T17:31:09.752Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
"A"
"B"
"C"
>>> Main thread finished with value: ()@{}%{}
